src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanJobService.java

```diff
@@ -29,17 +27,30 @@
   public static final String CREATED_BY_SYSTEM = "SYSTEM";
   public static final String NO_ACTIONPLAN_MSG = "ActionPlan not found for id %s";
 
-  @Autowired private DistributedLockManager actionPlanExecutionLockManager;
-
-  @Autowired private AppConfig appConfig;
-
-  @Autowired private ActionPlanRepository actionPlanRepo;
-
-  @Autowired private ActionCaseRepository actionCaseRepo;
-
-  @Autowired private ActionPlanJobRepository actionPlanJobRepo;
-
-  @Autowired private ActionService actionSvc;
+  private AppConfig appConfig;
+
+  private ActionCaseRepository actionCaseRepo;
+  private ActionPlanRepository actionPlanRepo;
+  private ActionPlanJobRepository actionPlanJobRepo;
```

Comment:
make all these member variables final

Reply:
Making these final was having strange effects on the tests. It seemed a bit challenging to fix the tests around it so leaving the `final` out for now